<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FfmpegPlayer</class>
 <widget class="QWidget" name="FfmpegPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="lbImageOrigin">
           <property name="text">
            <string>01</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbImageOriginT">
           <property name="text">
            <string>01</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="lbImageProcess">
           <property name="text">
            <string>01</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbImageProcessT">
           <property name="text">
            <string>01</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="labelVideoInfo">
       <property name="text">
        <string>--</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="pushButtonNextFrame_2">
         <property name="text">
          <string>Next frame</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btTimer">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cxCorrection">
         <property name="text">
          <string>Enabled image correction</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="pushButtonSeekFrame_2">
         <property name="text">
          <string>Seek to frame</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditFrame_2"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSeekMillisecond_2">
         <property name="text">
          <string>Seek to millisecond</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditMillisecond_2"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="cbOpenCV">
         <property name="text">
          <string>Use OpenCV</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btOpenVideo">
         <property name="text">
          <string>Open video</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
